src/server/index.js

```diff
@@ -33,6 +36,37 @@ async function createServer() {
   setupProxy()
   const redisClient = buildRedisClient(config.get('redis'))
 
+  const cacheProvider =
```

Comment:
I'll have a little look at this in the morning. As mentioned I'm not a fan of test code in production work. Will work something out